library/modules/bigip_software_install.py

```diff
@@ -24,6 +24,11 @@
     description:
       - Image to install on the remote device.
     type: str
+  block_device_image:
+    description:
+      - Image to install on the remote device.
+    type: str
+  version_added: "f5_modules 1.2"
```

Comment:
this is in the wrong spot make this in line with 'description' this is probably causing the sanity tests to collapse